packages/reactive-dict/reactive-dict.js

```diff
@@ -124,6 +139,47 @@ _.extend(ReactiveDict.prototype, {
     if (_.has(self.keys, key)) oldValue = parse(self.keys[key]);
     return EJSON.equals(oldValue, value);
   },
+  clear: function(key){
+    var self = this;
+    self.set(key, null);
+    return true;
+  },
+  toggle: function(key){
+    var self = this;
+
+    // toggle currently only works on boolean values
```

Comment:
I think there's a decision to be made here. IMO it would make sense for toggle to work just like `Session.set("x", ! Session.get("x"))`, since that's the pattern we are trying to replace?